packages/browser-wallet/src/popup/shared/Form/Input/Input.tsx

```diff
@@ -34,6 +34,7 @@ export const Input = forwardRef<HTMLInputElement, Props>(
                     ref={ref}
                     autoComplete="off"
                     spellCheck="false"
+                    readOnly={readOnly}
                     autoFocus={autoFocus}
```

Comment:
No need to explicitally add these; they would be included in the props spread right below.